ng/src/web/components/form/selectelements.js

```diff
@@ -46,6 +46,11 @@ export const Box = glamorous.div({
 } : null);
 
 export const Input = glamorous.input({
+  /* use font and line settings from parents not from browser default */
+  fontFamily: 'inherit',
+  fontSize: 'inherit',
+  lineHeight: 'inherit',
+
```

Comment:
Just curious why is this necessary?

Reply:
The old .css had those three settings for all ```<input>```, ```<textarea>``` and ```<select>``` elements. To not lose these, I added them here.

Reply:
Yeah but the original css was derived from bootstrap and relied on other css formattings. I am really not sure if that is necessary at all anymore. Even I am unsure about the original purpose.

Reply:
Leaving it out didn't make an obvious difference for the formgroups whose function I checked during programming. But then again, I didn't check all dialogs and use cases of the formgroups especially not with changed fonts etc. I can easily just delete it. Those settings are copied to some other files of this PR as well. Should I also delete those?